.chloggen/mx-psi_configretry.yaml

```diff
@@ -0,0 +1,26 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
Should this change just be mentioned in `.chloggen/addretrysetvalidation.yaml`? It's not a bugfix, right?

Reply:
Fixed